api/app/health.py

```diff
@@ -22,7 +22,7 @@ def patroni_cluster_health_check():
     header = {
         'Authorization': 'Bearer ' + config.get('STATUS_CHECKER_SECRET')
     }
-    resp = requests.get(url, headers=header)
+    resp = requests.get(url, headers=header, timeout=10)
```

Comment:
just wanted to double-check that the units for timeout are seconds, not milliseconds? (bc I was kinda expecting it would be ms for consistency with other cases where time is used, but for a timeout, seconds makes more sense than milliseconds)

Reply:
Consistent? Python? It's seconds - from the source code - lib/python3.8/site-packages/requests/api.py:

":param timeout: (optional) How many seconds to wait for the server to send data
        before giving up, as a float, or a :ref:`(connect timeout, read
        timeout) <timeouts>` tuple."

Reply:
Shall I throw in a comment?